crypto/src/main/scala/coop/rchain/crypto/util/CertificateHelper.scala

```diff
@@ -107,6 +107,7 @@ object CertificateHelper {
     info.set(X509CertInfo.ALGORITHM_ID, new CertificateAlgorithmId(algorithmId))
 
     // Sign the cert to identify the algorithm that's used.
+    @SuppressWarnings(Array("org.wartremover.warts.Var"))
     var cert = new X509CertImpl(info)
```

Comment:
This one can be easily fixed by declaring `cert2` in line 117.

Reply:
👍